### PR TITLE
Features/442 to 446

### LINKED ISSUE #442: Custom Toolbar should use only standard material ui theme atriubutes
Having very specific theme references makes the component hard to reuse.

Please consider replacing:
https://github.com/MetaCell/geppetto-meta/blob/master/geppetto.js/geppetto-ui/src/common/CustomToolbar.js#L14-L16

with components props

### PR DESCRIPTION
**Pull request by afonsobspinto**
closes #442 #443 #444 #446

### CODE REVIEW

File: geppetto.js/geppetto-ui/src/camera-controls/CameraControls.js [hunk 1]
```diff
@@ -143,14 +140,18 @@ class CameraControls extends Component {
         icon: null,
       });
     }
+
+    const buttonStyles = { color: '#fc6320', }
```
Comment (ddelpiano) [inline geppetto.js/geppetto-ui/src/camera-controls/CameraControls.js:144]: this does not work right now, it is hardcoded for geppetto but applications are customising the color of these controls

**Review by ddelpiano — CHANGES_REQUESTED** [review]

**Comment by ddelpiano** [comment]
@enicolasgomez this is a breaking change (tested in netpyne), imho it should go in a major release.
This is required for new applications that are now using material ui 5 but force who is using material ui 4 to adapt to this.
Thx

File: geppetto.js/geppetto-ui/src/camera-controls/CameraControls.js (hunk 1 above)
Comment (afonsobspinto) [reply geppetto.js/geppetto-ui/src/camera-controls/CameraControls.js:144]: Correct, this should be customizable through props. Thanks for the review :+1:

**Comment by afonsobspinto** [comment]
If the issues solved are in different milestones it will be beneficial to split the PR into multiple PR's (ideally one per issue, but at least one per milestone)

**Pull request closed without merging**